Sofya_Huliuk_Application/Program.cs

```diff
@@ -294,8 +294,161 @@ public void Task_9()
         uint d = (uint)b;
     }
 
+    public void Task_10()
+    {
+
+    }
+
 
 
 
 
 }
+
+class Homework_3
+{
+    public void Task_1()
+    {
+        while (true)
+        {
+            Console.WriteLine("Enter the integer:");
+            string input = Console.ReadLine();
+            bool result = int.TryParse(input, out var num);
+            if (!result)
+
+            {
+                Console.WriteLine("incorrect value, please try again");
+                continue;
+            }
+
+            if (num % 10 == 0)
+            {
+                Console.WriteLine("tutti-frutti");
+
+                continue;
+            }
+
+            if (num % 2 == 0)
+            {
+                Console.WriteLine("tutti");
+
+                continue;
+            }
+
+            if (num % 5 == 0)
+            {
+                Console.WriteLine("frutti");
+
+                continue;
+            }
+        }
+
+    }
+    public void Task_2()
+    {
+        Console.WriteLine("Enter the first number:");
+        string input1 = Console.ReadLine();
```

Comment:
according to the code style guide it is a bad habit to add numbers into the variable names

Reply:
your soft skills are impressive, but I will take this remark into account. tell me which variable name would be better in this case